magento/pwa-studio
# Fix apiBase URL in resolveUnknownRoute.js

## Description

_This pull request implements changes suggested by @mhaack in https://github.com/magento/pwa-studio/issues/2876._

Right now, it is possible to configure custom GraphQL endpoint by changing `apiBase`, queries made via Apollo work, but URL resolution in `resolveUnknownRoute.js` does not. This PR fixes resolveUnknownRoute to take apiBase into account.

## Related Issue

Closes #2876 
Closes [PWA-1137](https://jira.corp.magento.com/browse/PWA-1137)

## Acceptance




### Verification Stakeholders


### Specification



### Verification Steps

1. Change `apiBase` in `index.js` of venica-concept to point to a different endpoint
2. Verify GraphQL queries to resolve unknown URL are made to a different endpoint

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
